sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -246,11 +246,15 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 
 	sensorEvents := make([]*central.SensorEvent, 0, len(dockerConfig)+1)
 	registries := make([]*storage.ImagePullSecret_Registry, 0, len(dockerConfig))
+
+	saName := secret.GetAnnotations()[saAnnotation]
+	hasSAAnnotation := len(saName) > 0
```

Comment:
do we need a separate var for this or can we just check `saName != ""`. I generally prefer this over the len check as a nit

Reply:
changed, good to know the convention 👍